commitlint.config.js

```diff
@@ -44,6 +45,7 @@ module.exports = {
         'revert'
       ]
     ],
-    'type-case': [0]
+    'type-case': [0],
+    'header-max-length': [0, 'always', 150] // commit message first field (subject) length
```

Comment:
this is actually a second check for character limit. Seems this was taken very seriously